tests/test_PR.py

```diff
@@ -6,11 +6,19 @@
 import pytest
 from moviepy.video.fx.scroll import scroll
 from moviepy.video.io.VideoFileClip import VideoFileClip
+from moviepy.audio.io.AudioFileClip import AudioFileClip
 from moviepy.video.tools.interpolators import Trajectory
 from moviepy.video.VideoClip import ColorClip, ImageClip, TextClip
 from moviepy.video.compositing.CompositeVideoClip import CompositeVideoClip
+from moviepy.video.tools.subtitles import SubtitlesClip
 from moviepy.utils import close_all_clips
 
+try:
+    from pathlib import Path
+except ImportError:
+    Path = str
```

Comment:
There is a backport of Python3 [pathlib](https://pypi.org/project/pathlib2/) for python 2. Could you remove this (line 16 to 19) and add pathlib2 as dependency ? This way your code will be active even on Python2